pkg/log/default.go

```diff
@@ -52,6 +53,12 @@ func ErrorEnabled() bool {
 	return defaultScope.Level >= ErrorLevel
 }
 
+// Fatalf uses Errorf to construct and log a message at error level, and then exit.
+func Fatalf(template string, args ...interface{}) {
```

Comment:
For uniformity, all variants of the logging functions should be added (Fatal, Fatalw, Fatala). You also need to add equivalent functions on the Scope object.